spec/arke/exchange/binance_spec.rb

```diff
@@ -105,4 +116,30 @@
       end
     end
   end
+
+  context 'order_create' do
+    let(:order) { Arke::Order.new('ETHUSDT', '250', '1', :buy) }
+    let(:timestamp) { "1551720218" }
+    let(:query) do
+      "symbol=#{order.market}&side=#{order.side.upcase}&type=LIMIT&timeInForce=GTC&quantity=#{order.amount.to_f}"\
+        "&price=#{order.price.to_f}&recvWindow=5000&timestamp=#{timestamp}"
+    end
+    let(:signature) do
+      OpenSSL::HMAC.hexdigest(OpenSSL::Digest.new('sha256'), 'OwpadzSYOSkzweoJkjPrFeVgjOwOuxVHk8FXIlffdWw', query)
+    end
+
```

Comment:
are you using webmock ? where do you mock the binance api ?

Reply:
in next file, `mocked_context.rb`